model_analyzer/constants.py

```diff
@@ -62,3 +62,6 @@
 
 # Constraints
 GLOBAL_CONSTRAINTS_KEY = "__default__"
+
+# Measurement threshold
```

Comment:
This comment means nothing. Either expand it or remove it.

Reply:
Fixed